UnityProject/Assets/Scripts/Shuttles/EscapeShuttle.cs

```diff
@@ -97,6 +78,47 @@ public int InitialTimerSeconds
 	/// </summary>
 	public int CurrentTimerSeconds { get; private set; }
 
+	public int GiveUpTime { get; private set; } = 200;
+
+	/// <summary>
+	/// Current "flight" time
+	/// </summary>
+	public int UnderflowIndex { get; private set; }
+
+	public List<string> UnderflowFunnies = new List<string>()
```

Comment:
huh?

Reply:
https://www.youtube.com/watch?v=dzMq5_thk4o&t=75s

will only show if the shuttle is taking longer than the countdown